mwsql-blogpost.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<blockquote>In just two lines of code, we have created a <code>Dump</code> object that contains all the data from the file. Well, not exactly. If we were working with a very large file -- and most of the dump files <em>are</em> indeed very large -- we would quickly run out of memory if we tried ingesting all the file contents at once. </blockquote><blockquote><br></blockquote><blockquote>In reality, most of the parsing happens only when we explicitly request the rows, or records, of the table.  The file contents are processed sequentially, holding only parts of the file in memory. No matter how large the file is, we will not run out of RAM.  </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/blancadesal/wm-tutorials-and-blogposts/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>